### PR TITLE
Reply to DLC open/close offers immediately (and only once)

### PR DESCRIPTION
**Pull request by luckysori**
Fixes https://github.com/get10101/10101/issues/275.

### CODE REVIEW

File: crates/ln-dlc-node/src/node/dlc_channel.rs [hunk 1]
```diff
@@ -75,6 +75,24 @@ impl Node {
...
+        tracing::info!(channel_id = %channel_id_hex, "Rejecting DLC channel offer");
+
+        let (node_id, accept_sub_channel) = self
+            .sub_channel_manager
+            .accept_sub_channel(channel_id)
+            .map_err(|e| anyhow!(e.to_string()))?;
+
+        self.dlc_message_handler.send_message(
+            node_id,
+            Message::SubChannel(SubChannelMessage::Accept(accept_sub_channel)),
```
Comment (holzeis) [inline crates/ln-dlc-node/src/node/dlc_channel.rs:90]: 🤔 This looks strange. Is `Accept` really the sub channel message that we are supposed to send here? 😅

File: mobile/native/src/ln_dlc/node.rs [hunk 2]
```diff
@@ -36,78 +33,107 @@ impl Node {
...
+                        }
+                    }
+
+                    if let Some(reply_msg) = reply_msg {
+                        tracing::debug!(
+                            to = %node_id,
+                            msg = %sub_channel_message_as_str(&reply_msg),
+                            "Sending DLC channel message"
+                        );
+                        self.inner
```
Comment (holzeis) [inline mobile/native/src/ln_dlc/node.rs:99]: side note, we will have to delay the sending of the confirm messages here.

**Review by holzeis — APPROVED** [review]
Nice :) 👍

File: crates/ln-dlc-node/src/node/dlc_channel.rs (hunk 1 above)
Comment (luckysori) [reply crates/ln-dlc-node/src/node/dlc_channel.rs:90]: Oh, good catch. I was investigating if there was a way to reject subchannel open/close offers (there is none currently) and left this hanging around by mistake. It's not used and it's obviously wrong, so I'll remove it.

File: mobile/native/src/ln_dlc/node.rs (hunk 2 above)
Comment (luckysori) [reply mobile/native/src/ln_dlc/node.rs:99]: Confirm is sent by the coordinator though.

**Pull request merged**